presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableWriterNode.java

```diff
@@ -216,6 +218,9 @@ public String toString()
         {
             return catalog + "." + tableMetadata.getTable();
         }
+
+        @Override
+        public String toPlanString() { return toString(); }
```

Comment:
Let's provide explicit implementation to avoid having a dependency on `toString`